persistence-api/src/main/java/io/stargate/db/schema/Keyspace.java

```diff
@@ -29,6 +29,8 @@ public abstract class Keyspace implements SchemaEntity {
 
   private static final long serialVersionUID = -337891773492616286L;
 
+  private int schemaHashCode; // Lazily loaded schema hash code
```

Comment:
You can use `@org.immutables.value.Value.Derived` on the `schemaHashCode()` for eager calculation and caching during the construction time, and thus remove the logic added.. Or `@Lazy` is you prefer lazy computation, but that has some sync stuff that we could avoid here..

Reply:
Oh, nice, good thought!